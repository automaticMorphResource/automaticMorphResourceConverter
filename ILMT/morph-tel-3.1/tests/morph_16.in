1	puttagoVdugulawo	unk	
1	guMdeV	unk	
1	paxilaM	unk	
1	.	unk	
