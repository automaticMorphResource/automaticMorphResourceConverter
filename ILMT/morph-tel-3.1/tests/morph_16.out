1	puttagoVdugulawo	unk	<fs af='puttagoVdugu,n,,pl,,,wo,wo'>
2	guMdeV	unk	<fs af='guMdeV,n,,sg,,d,0,0'>|<fs af='guMdeV,n,,sg,,o,ti,ti'>
3	paxilaM	unk	<fs af='paxilaM,n,,sg,,d,0,0'>
4	.	unk	<fs af='&dot;,punc,,,,,,'>
